src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,52 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private Engine engineOfCar;
+    private String mark;
```

Comment:
```suggestion
    private String manufacturer;
```
change getter, setter, constructor and toString() accordingly